x/concentrated-liquidity/internal/math/math_test.go

```diff
@@ -10,6 +10,13 @@ import (
 	"github.com/osmosis-labs/osmosis/v13/x/concentrated-liquidity/internal/math"
 )
 
+var (
+	MaxSpotPrice                   = sdk.MustNewDecFromStr("100000000000000000000000000000000000000")
+	MinSpotPrice                   = sdk.MustNewDecFromStr("0.000000000000000001")
```

Comment:
```suggestion
        // TODO: Comment on difference or otherwise align with standards elsewhere in codebase
	MaxSpotPrice                   = sdk.MustNewDecFromStr("100000000000000000000000000000000000000")
	MinSpotPrice                   = sdk.MustNewDecFromStr("0.000000000000000001")
```